rest-api-spec/src/main/resources/rest-api-spec/test/index/100_partial_flat_object.yml

```diff
@@ -88,7 +88,16 @@ setup:
             } ]
           }
         }
-
+  - do:
```

Comment:
I was wondering why things *didn't* fail on BWC tests already (since the fix hasn't been backported).

All the tests in this file are currently marked `skip: version: " - 2.99.99"`, so they haven't been part of the BWC tests.

Can we apply this fix, then I'll make the change to update the skip version to `2.16.99` for these tests? (If I try to set it to 2.16.99 now, we'll break BWC until we backport.)

Reply:
Got it, thanks @msfroh !